source/transactions-convenient-api/transactions-convenient-api.md

```diff
@@ -128,11 +129,23 @@ This method should perform the following sequence of actions:
 6. If the callback reported an error:
     1. If the ClientSession is in the "starting transaction" or "transaction in progress" state, invoke
         [abortTransaction](../transactions/transactions.md#aborttransaction) on the session.
+
     2. If the callback's error includes a "TransientTransactionError" label and the elapsed time of `withTransaction` is
-        less than 120 seconds, jump back to step two.
+        less than 120 seconds, calculate the backoffMS to be
+        `jitter * min(BACKOFF_INITIAL * (1.25**retry), BACKOFF_MAX)` where:
+
+        1. jitter is a random float between \[0, 1)
+        2. retry is one less than the number of times Step 2 has been executed since Step 1 was executed
```

Comment:
This phrasing reads odd to me.  Maybe we can say something like:

```suggestion
        2. retry is the retry number minus one (the first retry uses `retry=0`)
```

Thoughts?  Open to other phrasing ideas too.

Reply:
Actually, after looking at the pseudocode below, would it make sense to treat `retry` the same way that `startTime` is treated?  Concretely:

- Step 1 in this list would mention setting retry to 0
- Step 6 would mention incrementing retry after the backoff

Reply:
I like treating `retry` similar to `startTime` its much clearer and more representative of how the code is likely going to be written. changed in [bc9153e](https://github.com/mongodb/specifications/pull/1851/commits/bc9153eb4375e0b3a77ba8ecccd96af4432c3907)